### ISSUE TITLE
About the results

### ISSUE THREAD

**Issue by kwonmha** [Soptq/LANA-pytorch#1]
Hello, thanks for opening codes for the paper.

I found that there's no codes related to processing and testing with test set.
Are the performance results you reported obtained from valid dataset?

**Comment by Soptq** [comment]
Yes, we are actually using validation set as test set in our experiments.

**Comment by kwonmha** [comment]
Thanks for informing.
